ocs_ci/ocs/ui/views.py

```diff
@@ -228,6 +228,11 @@
     "check_first_row_checkbox": ('input[name="checkrow0"]', By.CSS_SELECTOR),
     "remove_search_filter": ('button[aria-label="close"]', By.CSS_SELECTOR),
     "delete_resource_kebab_button": ('//*[contains(text(), "Delete")]', By.XPATH),
+    "ocs-storagecluster-storagesystem": (
+        '//a[text()="ocs-storagecluster-storagesystem"]',
+        By.XPATH,
+    ),
+    "block_pool_tab": ('//span[normalize-space()="BlockPools"]', By.XPATH),
```

Comment:
we already have this locator
views.py -> validation_4_11. 
It should be available from self.validation_loc["blockpools"]

Reply:
Made the changes.